src/shared/logs-show.c

```diff
@@ -500,7 +489,7 @@ static int output_verbose(
         else {
                 _cleanup_free_ char *value = NULL;
 
-                r = parse_field(data, length, "_SOURCE_REALTIME_TIMESTAMP=", &value, NULL);
+                r = parse_field_single(data, length, "_SOURCE_REALTIME_TIMESTAMP=", 27, &value, NULL);
```

Comment:
Please use `strlen( "_SOURCE_REALTIME_TIMESTAMP=")` instead. It is replaced by the actual length even at -O0.